.travis.yml

```diff
@@ -0,0 +1,76 @@
+env:
+  global:
+    - LC_CTYPE=en_US.UTF-8
+language: objective-c
+matrix:
+  include:
+    - osx_image: xcode7.3
+      env:
+        - XCODE_SDK=macosx
+          XCODE_ACTION=test
+          XCODE_DESTINATION="arch=x86_64"
+    - osx_image: xcode7.3
+      env:
+        - XCODE_SDK=iphonesimulator
+          XCODE_ACTION=test
+          XCODE_DESTINATION="name=iPhone 6s"
+    - osx_image: xcode7.3
+      env:
+        - XCODE_SDK=appletvsimulator
+          XCODE_ACTION=test
+          XCODE_DESTINATION="name=Apple TV 1080p"
+    - osx_image: xcode7.3
+      env:
+        - XCODE_SDK=watchsimulator
+          XCODE_ACTION=build
+          XCODE_DESTINATION="name=Apple Watch - 38mm"
+    - osx_image: xcode7.3
+      env:
+        - JOB=CARTHAGE
+      script:
+        - brew update
+        - brew outdated carthage || brew upgrade carthage
+        - carthage build --no-skip-current
+        - for platform in Mac iOS tvOS watchOS; do test -d Carthage/Build/$platform/PrettyColors.framework || exit 1; done
+    - osx_image: xcode8
+      env:
+        - XCODE_SDK=macosx
+          XCODE_ACTION=test
+          XCODE_DESTINATION="arch=x86_64"
+    - osx_image: xcode8
+      env:
+        - XCODE_SDK=iphonesimulator
+          XCODE_ACTION="build-for-testing test-without-building"
+          XCODE_DESTINATION="name=iPhone 6s"
+    - osx_image: xcode8
+      env:
+        - XCODE_SDK=appletvsimulator
+          XCODE_ACTION="build-for-testing test-without-building"
+          XCODE_DESTINATION="name=Apple TV 1080p"
+    - osx_image: xcode8
+      env:
+        - XCODE_SDK=watchsimulator
+          XCODE_ACTION=build
+          XCODE_DESTINATION="name=Apple Watch - 38mm"
+    - osx_image: xcode8
+      env:
+        - JOB=CARTHAGE
+      script:
+        - brew update
+        - brew outdated carthage || brew upgrade carthage
+        - carthage build --no-skip-current
+        - for platform in Mac iOS tvOS watchOS; do test -d Carthage/Build/$platform/PrettyColors.framework || exit 1; done
+git:
+  submodules: false
+before_install:
+  - git submodule update --init --recursive
```

Comment:
Curious about the above lines, assuming they're a workaround for some issue?

I wouldn't be concerned about them, but jobs are failing with `fatal: unable to access 'https://github.com/jdhealy/FABConfig.git/': Failed to connect to github.com port 443: Operation timed out`: https://travis-ci.org/jdhealy/PrettyColors/jobs/148949181#L281 — possibly getting throttled by GitHub?